src/main/java/com/google/api/codegen/ApiaryConfig.java

```diff
@@ -230,4 +244,29 @@ public Field getField(Type type, String fieldName) {
   public ListMultimap<String, String> getAuthScopes() {
```

Comment:
QQ: Is it customary to return the ListMultimap itself, or is it bett to have getAuthScopes(method) return the list of scopes for that method?